pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -396,16 +399,37 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 
 	var nsesReg [nodesCount]*registry.NetworkServiceEndpoint
 	for i := 0; i < nodesCount; i++ {
+		requestCount := 0
 		var additionalFunctionality []networkservice.NetworkServiceServer
 		if i != 0 {
-			// Passtrough to the node i-1
+			connCtx := &networkservice.ConnectionContext{
+				ExtraContext: map[string]string{
+					fmt.Sprintf("testkey-%v", i): fmt.Sprintf("testvalue-%v", i),
+				},
+				IpContext: &networkservice.IPContext{},
+			}
+
+			// Pass through to the node i-1
 			additionalFunctionality = []networkservice.NetworkServiceServer{
 				chain.NewNetworkServiceServer(
 					clienturl.NewServer(s.domain.Nodes[i].NSMgr.URL),
+					checkrequest.NewServer(t, func(t *testing.T, request *networkservice.NetworkServiceRequest) {
+						if requestCount > 0 {
+							assert.Equal(
```

Comment:
Is there a reason to use `assert` package instead on `require`?

Reply:
suggested by Bolodya1997

Reply:
I fully agree that we should have a check here. But I don't see Bolodya1997 suggesting to use `assert`.
`assert` package fails the test, but then the test continues. `require` package fails the test and aborts it.
I don't see why we would want to continue this test if the assertion failed.